docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,589 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "updated_isvc = V1beta1InferenceService(api_version= constants.KSERVE_V1BETA1,\n",
+    "                          kind=constants.KSERVE_KIND,\n",
+    "                          metadata=client.V1ObjectMeta(name=name, namespace=namespace),\n",
+    "                          spec=updated_spec)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'apiVersion': 'serving.kserve.io/v1beta1',\n",
+       " 'kind': 'InferenceService',\n",
+       " 'metadata': {'annotations': {'serving.kserve.io/deploymentMode': 'ModelMesh'},\n",
+       "  'creationTimestamp': '2023-06-09T21:41:14Z',\n",
+       "  'generation': 2,\n",
+       "  'managedFields': [{'apiVersion': 'serving.kserve.io/v1beta1',\n",
+       "    'fieldsType': 'FieldsV1',\n",
+       "    'fieldsV1': {'f:status': {'.': {},\n",
+       "      'f:components': {'.': {},\n",
+       "       'f:predictor': {'.': {},\n",
+       "        'f:grpcUrl': {},\n",
+       "        'f:restUrl': {},\n",
+       "        'f:url': {}}},\n",
+       "      'f:conditions': {},\n",
+       "      'f:modelStatus': {'.': {},\n",
+       "       'f:copies': {'.': {}, 'f:failedCopies': {}, 'f:totalCopies': {}},\n",
+       "       'f:states': {'.': {},\n",
+       "        'f:activeModelState': {},\n",
+       "        'f:targetModelState': {}},\n",
+       "       'f:transitionStatus': {}},\n",
+       "      'f:url': {}}},\n",
+       "    'manager': 'manager',\n",
+       "    'operation': 'Update',\n",
+       "    'subresource': 'status',\n",
+       "    'time': '2023-06-09T21:41:29Z'},\n",
+       "   {'apiVersion': 'serving.kserve.io/v1beta1',\n",
+       "    'fieldsType': 'FieldsV1',\n",
+       "    'fieldsV1': {'f:metadata': {'f:annotations': {'.': {},\n",
+       "       'f:serving.kserve.io/deploymentMode': {}}},\n",
+       "     'f:spec': {'.': {},\n",
+       "      'f:predictor': {'.': {},\n",
+       "       'f:sklearn': {'.': {},\n",
+       "        'f:name': {},\n",
+       "        'f:protocolVersion': {},\n",
+       "        'f:storage': {'.': {}, 'f:key': {}, 'f:path': {}}},\n",
+       "       'f:tensorflow': {'.': {},\n",
+       "        'f:name': {},\n",
+       "        'f:protocolVersion': {},\n",
+       "        'f:storage': {'.': {}, 'f:key': {}, 'f:path': {}}}}}},\n",
+       "    'manager': 'OpenAPI-Generator',\n",
+       "    'operation': 'Update',\n",
+       "    'time': '2023-06-09T21:41:54Z'}],\n",
+       "  'name': 'msnist-sample',\n",
+       "  'namespace': 'modelmesh-serving',\n",
+       "  'resourceVersion': '4014187',\n",
+       "  'uid': '4360f17b-3575-4b0d-a324-3ba536b55697'},\n",
+       " 'spec': {'predictor': {'sklearn': {'name': '',\n",
+       "    'protocolVersion': 'v2',\n",
+       "    'storage': {'key': 'localMinIO', 'path': 'sklearn/mnist-svm.joblib'}},\n",
```

Comment:
I noticed that `sklearn/mnist-svm.joblib` still appears along with `tensorflow/mnist.savedmodel` below in the same spec. It seems like in this case you'd actually want to use the `replace` as explained [here](https://kserve.github.io/website/0.10/sdk_docs/docs/KServeClient/#replace), instead of `patch`.